commands/internal/setpipelinehelpers/atc_config.go

```diff
@@ -83,8 +83,6 @@ func (atcConfig ATCConfig) Validate(
 		displayhelpers.Failf("configuration invalid")
 	}
 
-	fmt.Println("looks good")
```

Comment:
this shouldn't be removed

Reply:
oh my bad... it is "looks good" for validation.